azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/Pool.java

```diff
@@ -30,14 +31,79 @@
      */
     List<MountConfiguration> mountConfiguration();
 
+    /**
+     * @return the scaleSettings value
+     */
+    ScaleSettings scaleSettings();
+
+    /**
+     * @return the startTask value
+     */
+    StartTask startTask();
+
+    /**
+     * @return the metadata value
+     */
+    List<MetadataItem> metadata();
+
+    /**
+     * @return the applicationPackages value
+     */
+    List<ApplicationPackageReference> applicationPackages();
+
+    /**
+     * @return the certificates value
+     */
+    List<CertificateReference> certificates();
+
+    /**
+     * @return the size of virtual machine in the pool

```

Comment:
Wondering why the size is string but not scalar value?

Reply:
The vmSize will have value like "STANDARD_D4" 
[https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/examples/PoolCreate_FullExample.json#L11](url)